apps/framework-cli/deploy/kubernetes/config-files/moose-deployment.yaml

```diff
@@ -107,5 +107,7 @@ spec:
                   key: sk-moose-http-server-config--port
             - name: MOOSE_LOGGER_STDOUT
               value: "true"
+            - name: MOOSE_INTERNAL
+              value: "true"
```

Comment:
I noticed a lot of `moose prod` commands when we were still sending to UserActivity. I think it's coming from our internal moose, so figured we can add an env variable to not collect internal activities. We can probably use the same env var on our local dev envs.